### PR TITLE
log db statement with parameter value when use SmartSql

### PR DESCRIPTION
**Pull request by xclw2000**
Please answer these questions before submitting pull request

- Why submit this pull request?
- New feature provided

- Related issues
Can't log parameter values

___
### New feature or improvement
- log parameter values when log db statement

### CODE REVIEW

File: src/SkyApm.Diagnostics.SmartSql/SmartSqlTracingDiagnosticProcessor.cs [hunk 1]
```diff
@@ -249,7 +250,12 @@ public void BeforeCommandExecuterExecute([Object]CommandExecuterExecuteBeforeEve
             var context = CreateSmartSqlLocalSegmentContext(eventData.Operation);
             if (eventData.ExecutionContext.Request.RealSql != null)
             {
-                context.Span.AddTag(Common.Tags.DB_STATEMENT, eventData.ExecutionContext.Request.RealSql);
+                //context.Span.AddTag(Common.Tags.DB_STATEMENT, eventData.ExecutionContext.Request.RealSql);
```
Comment (wu-sheng) [inline src/SkyApm.Diagnostics.SmartSql/SmartSqlTracingDiagnosticProcessor.cs:253]: Why do you comment this line?

Comment (xclw2000) [reply src/SkyApm.Diagnostics.SmartSql/SmartSqlTracingDiagnosticProcessor.cs:253]: Below varialbe "sql" include "RealSql"
for example:
ReqlSql="select 1 from T where F1=@F1"
sql="selece 1 from T where F1='aaa'"

Comment (wu-sheng) [reply src/SkyApm.Diagnostics.SmartSql/SmartSqlTracingDiagnosticProcessor.cs:253]: I mean if you try to delete this, delete this. Git is the time machine. We don't need to keep old codes.

Comment (xclw2000) [reply src/SkyApm.Diagnostics.SmartSql/SmartSqlTracingDiagnosticProcessor.cs:253]: Recommit pr after I delete it?

Comment (wu-sheng) [reply src/SkyApm.Diagnostics.SmartSql/SmartSqlTracingDiagnosticProcessor.cs]: Yes, if your plan is to remove this line.

Comment (xclw2000) [reply src/SkyApm.Diagnostics.SmartSql/SmartSqlTracingDiagnosticProcessor.cs]: done!

**Review by liuhaoyang — APPROVED** [review]

**Comment by liuhaoyang** [comment]
> Let's wait for @liuhaoyang to review other details.

LGTM

**Pull request merged**